src/components/DocumentationTopic/TopicsLinkCardGrid.vue

```diff
@@ -0,0 +1,47 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="TopicsLinkCardGrid">
+    <Row :columns="compactCards ? 3 : 2">
+      <Column
+        v-for="(item, index) in items"
+        :key="index"
+      >
+        <TopicsLinkCardGridItem :item="item" :compact="compactCards" />
+      </Column>
+    </Row>
+  </div>
+</template>
+
+<script>
+import Row from 'docc-render/components/ContentNode/Row.vue';
+import Column from 'docc-render/components/ContentNode/Column.vue';
+import { TopicStyles } from 'docc-render/constants/TopicStyles';
+import TopicsLinkCardGridItem from './TopicsLinkCardGridItem.vue';
+
+export default {
+  name: 'TopicsLinkCardGrid',
+  components: { TopicsLinkCardGridItem, Column, Row },
+  props: {
+    items: {
+      type: Array,
+      required: true,
+    },
+    topicStyle: {
```

Comment:
You may want to add a validator to ensure that only `TopicStyles.compactGrid` and `TopicStyles.detailedGrid` are valid strings here (assuming those are the only 2)—and, if those are the only 2, maybe it would be even better to turn this into a boolean `compact` or `detailed` prop.